### ISSUE TITLE
pod name not recognised by kubectl

### ISSUE THREAD

**Issue by kiwiidb** [kubermatic/fubectl#42]
Hi there, thanks for the great tool.

I have used it for the past year and recently started using it again on a new computer.

Now, whenever I use `klog` or `kfor`, it fails with the following error message:
```
error: expected 'logs [-f] [-p] (POD | TYPE/NAME) [-c CONTAINER]'.
POD or TYPE/NAME is a required argument for the logs command
See 'kubectl logs -h' for help and examples
```
Any idea why this could be?

Separating the namespace and pod name solves the issue:
https://github.com/kiwiidb/fubectl/commit/282bc633ba32516788cacf6a546a483618a6e1dc

**Comment by KevinDenys** [comment]
I also have this issue

**Comment by KevinDenys** [comment]
@kiwiidb add setopt SH_WORD_SPLIT to the end of the source file and try again.

**Comment by scheeles** [comment]
closed by #53
